scripts/docker/dockerfile_clang-10

```diff
@@ -22,14 +18,15 @@ USER serac
 RUN git clone --recursive --branch $branch --single-branch --depth 1 https://github.com/LLNL/serac.git
 
 # New containers will use Python3
-RUN cd serac && python3 ./scripts/uberenv/uberenv.py --spack-config-dir=./scripts/spack/configs/docker/ubuntu18/ \
-                                                     --project-json=.uberenv_config.json \
-                                                     --spec=%clang@10.0.0 --prefix=/home/serac/serac_tpls -k \
-             && rm -rf /home/serac/serac_tpls/builds
+RUN cd serac && python3 ./scripts/uberenv/uberenv.py --spack-config-dir=./scripts/spack/configs/docker/ubuntu20/ \
+                                                       --project-json=.uberenv_config.json \
+                                                       --spec=%clang@10.0.0 --prefix=/home/serac/serac_tpls -k
+# white238: commented out due to logs disapearing and this being saved due to layering
```

Comment:
This was probably wrong in hindsight but did make the problem go away if i remember correctly